README.md

```diff
@@ -18,7 +18,15 @@ Install node database driver(s) of the database(s) that are to be synced in your
 
     $ yarn add mssql
 
-### Configure Database Connections
+### Global
```

Comment:
No need another section for this. You can just add the line below the Installation section with a short note for global installation.